index.html

```diff
@@ -0,0 +1,188 @@
+                </div>
+                <div style="clear:both;"></div>
+            </div>
+        </section>
+        <section>
+            <div class="container">
+                <div class="two-columns-grid">
+                    <div class="two-columns-grid-item">
+                        <div class="three-rows">
+                            <div class="three-rows-item">
+                                <div class="three-rows-item-img">
+                                    <img src="https://via.placeholder.com/100x100" alt="test">
+                                </div>
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium quam rerum dolorem delectus soluta ipsa eius quas, unde labore animi ratione nemo molestias fugit magni odio dolorum suscipit hic nobis provident facilis placeat eveniet impedit! Deleniti aliquam modi.</p>
+                            </div>
+                            <div class="three-rows-item">
+                                <div class="three-rows-item-img">
+                                    <img src="https://via.placeholder.com/100x100" alt="test">
+                                </div>
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium quam rerum dolorem delectus soluta ipsa eius quas, unde labore animi ratione nemo molestias fugit magni odio dolorum suscipit hic nobis provident facilis placeat eveniet impedit! Deleniti aliquam modi.</p>
+                            </div>
+                            <div class="three-rows-item">
+                                <div class="three-rows-item-img">
+                                    <img src="https://via.placeholder.com/100x100" alt="test">
+                                </div>
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium quam rerum dolorem delectus soluta ipsa eius quas, unde labore animi ratione nemo molestias fugit magni odio dolorum suscipit hic nobis provident facilis placeat eveniet impedit! Deleniti aliquam modi.</p>
+                            </div>
+                        </div>
+                    </div>
+                    <div style="clear: both;"></div>
```

Comment:
takich trikow sie juz raczej nie stosuje, wiem po co to dodales, bo Ci sie boxy z obrazkiem sypały, ogólnie zeby naprawic to co Ci sie tutaj psulo to wystarczy jak na ".three-rows-item" ustawisz "overflow: hidden;" a tego diva z cleare-em wywal